packages/jest-snapshot/src/State.ts

```diff
@@ -86,20 +86,24 @@ export default class SnapshotState {
   _addSnapshot(
     key: string,
     receivedSerialized: string,
-    options: {isInline: boolean, error?: Error},
+    options: {isInline: boolean; error?: Error},
   ) {
     this._dirty = true;
     if (options.isInline) {
       const error = options.error || new Error();
-      const lines = getStackTraceLines(error.stack);
+      const lines = getStackTraceLines(error.stack || '');
       const frame = getTopFrame(lines);
       if (!frame) {
         throw new Error(
           "Jest: Couldn't infer stack frame for inline snapshot.",
         );
       }
       this._inlineSnapshots.push({
-        frame,
+        frame: {
+          column: frame.column,
+          file: frame.file as string,
```

Comment:
why is the typecast needed?

Reply:
`stack-utils` returns following data format:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/b429791b8beb059ea61fd5b391b097ab55dc60d0/types/stack-utils/index.d.ts#L54-L58

However `jest-message-util` checks `file` field and returns only when `file` is defined:

https://github.com/facebook/jest/blob/fb1f32688b41a59f4318932ba1e8b55c4ba691ff/packages/jest-message-util/src/index.ts#L236-L244

Reply:
Ah, right! Seems we should fix the type in `jest-message-util`, then

Reply:
I will change then